.github/workflows/build-and-test.yml

```diff
@@ -60,10 +60,96 @@ jobs:
           name: dist
           path: dist/
 
+  test:
```

Comment:
I'll need to adjust the branch protections to require this job, not the previous one, so that this must run on all PRs